asv_bench/benchmarks/index_concat.py

```diff
@@ -0,0 +1,20 @@
+from .pandas_vb_common import *
+
+
+class Int64Indexing(object):
```

Comment:
Can you put this one in `index_object.py` ? And maybe call the class `IndexOps` or something (not 'indexing' as it append has nothing to do with indexing)